xbmc/Util.cpp

```diff
@@ -1025,7 +1025,9 @@ std::string CUtil::ValidatePath(const std::string &path, bool bFixDoubleSlashes
 
 bool CUtil::IsUsingTTFSubtitles()
 {
-  return URIUtils::HasExtension(CServiceBroker::GetSettingsComponent()->GetSettings()->GetString(CSettings::SETTING_SUBTITLES_FONT), ".ttf");
+  return URIUtils::HasExtension(CServiceBroker::GetSettingsComponent()->GetSettings()->GetString(
```

Comment:
This doesn't seem correct, otf is not ttf. So IsUsingTTFSubtitles should be false for otf. Maybe rename to SubtitlesHaveFontExtensions or similar